libmscore/paste.cpp

```diff
@@ -443,7 +447,7 @@ void Score::pasteChordRest(ChordRest* cr, int tick, const Interval& srcTranspose
 
       int measureEnd = measure->endTick();
       bool isGrace = (cr->type() == Element::Type::CHORD) && (((Chord*)cr)->noteType() != NoteType::NORMAL);
-      if (!isGrace && (tick + cr->actualTicks() > measureEnd || convertMeasureRest)) {
+      if (!isGrace && !cr->tuplet() && (tick + cr->actualTicks() > measureEnd || convertMeasureRest)) {
```

Comment:
Well, it's not that we "shouldn't be there" if cr is a tuplet.  If the last note of the tuplet extend to the barline, that's not a problem.  The problem is that actualTicks() might be a little too long, so without this check, it might *appear* that the note won't fit in the measure in though it really does.  Since we have already disallowed tuplets that cross barlines, it seems safe to simply skip this code for tuplets.  And yes, a comment is a good idea, am adding it.